build/utils.py

```diff
@@ -21,6 +21,24 @@
 # Third Party
 import torch
 from accelerate.commands.launch import launch_command_parser
+import shutil
+
+
+def copytree(source, destination):
```

Comment:
Can you include some of the unit tests you made for running this method in [test_utils](https://github.com/foundation-model-stack/fms-hf-tuning/blob/main/tests/build/test_utils.py). A few useful test cases would be:
- Creating a tmp directory with foo and bar files and then running copytree on it and ensuring those two files exist in destination
-  Creating a tmp directory with foo directory and bar inside like /tmp/foo/bar and then running copytree on it and ensuring dir and file exists in destination
- Ensure it works even if destination dir already exists
- Ensure it works with symlinks as needed

Reply:
Yes I have all of those already, [here](https://github.com/olson-ibm/copytree/blob/main/test/test.py).

I'll get them over ASAP.

Reply:
- Success case if dest dir does not exist
- Success case if dest dir does already exist
- Failure case if dest dir not writeable
- Failure case if source dir does not exist